yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/server/RunExternalServerExtension.java

```diff
@@ -87,7 +86,6 @@ public String getVersion() {
 
     @Override
     public void beforeAll(ExtensionContext context) throws Exception {
-        Assumptions.abort(); // Will be able to re-enable when we have a published external server to use here
```

Comment:
Given the issues with explain changing between versions, I'm not sure that we want to enable this as part of prb yet.

Reply:
Do you mean in general, or just for the tests that contain explain assertions?

Reply:
The issue was resolved by https://github.com/FoundationDB/fdb-record-layer/pull/3055